src/pages/Quiz/Quiz.js

```diff
@@ -8,20 +8,36 @@ import head from '../../public/img/RicksHead.png';
  */
 // eslint-disable-next-line no-unused-vars
 function Quiz(props) {
+  const allAnswers = [];
+  const { element: counterElement, increment, getCount } = QuestionCounter();
+  const onClick = () => {
+    const answeredQuestions = getCount() - 1;
+    const points = allAnswers.filter((answer) => answer.correct).length;
+    console.log(`Quiz finished after answering ${answeredQuestions} questions!`);
```

Comment:
Chyba bym usunął te console.log, wiem że to takie tymczasowe rozwiązanie, ale lepiej zeby na main to nie weszło, bo jeszcze zapomnimy aby usunąć.